src/decision/CMakeLists.txt

```diff
@@ -36,11 +37,35 @@ include_directories(
 
 ## Declare a C++ executable
 # add_executable(sample_package_node src/sample_package_node.cpp)
-add_executable(lidar_decision src/lidar_decision.cpp src/LidarDecision.cpp include/LidarDecision.h src/LidarObstacle.cpp include/LidarObstacle.h)
-add_executable(vision_decision src/vision_decision.cpp src/VisionDecision.cpp include/VisionDecision.h)
-add_executable(gps_decision src/gps_decision.cpp src/GpsDecision.cpp src/GpsMover.cpp include/GpsDecision.h include/GpsMover.h)
-add_executable(final_decision src/final_decision.cpp src/FinalDecision.cpp include/FinalDecision.h)
-add_executable(gps_manager src/gps_manager.cpp src/GpsManager.cpp include/GpsManager.h)
+add_executable(lidar_decision 
+    src/lidar_decision.cpp
+    src/LidarDecision.cpp
+    include/LidarDecision.h
+    src/LidarObstacle.cpp
+    include/LidarObstacle.h
+    )
+add_executable(vision_decision 
+    src/vision_decision.cpp 
+    src/VisionDecision.cpp
+    include/VisionDecision.h
+    )
+add_executable(gps_decision 
+    src/gps_decision.cpp 
+    src/GpsDecision.cpp
+    src/GpsMover.cpp
+    include/GpsDecision.h
+    include/GpsMover.h
+    )
+add_executable(final_decision 
+    src/final_decision.cpp
+    src/FinalDecision.cpp 
+    include/FinalDecision.h
+    )
+add_executable(gps_manager 
+    src/gps_manager.cpp
+    src/GpsManager.cpp 
+    include/GpsManager.h
+    )
```

Comment:
I'm thinking about setting this as standard formatting for all `CMakeLists.txt` files, but that's pretty hard to enforce, so not sure if it's feasible :cry: